src/style.css

```diff
@@ -1 +1,37 @@
-/* add styles here */
+/* #region html */
+body {
+  margin: 0;
+}
+
+/* #endregion */
+
+/* #region Stars Inactive */
+.stars {
```

Comment:
[checklist](https://github.com/mate-academy/layout_stars?tab=readme-ov-file#checklist): Each BEM block has its own separate file
create separate file for stars block